src/test/java/greencity/service/impl/PlaceServiceImplTest.java

```diff
@@ -73,54 +156,22 @@
 
     @Test
     public void save() {
-        Category category = setCategoryToPlaceTest();
-
-        Place place = new Place();
-        place.setName("test");
-        place.setCategory(category);
-        Place expectedPlace = new Place();
-        place.setId(1L);
-        place.setName("test");
-        place.setCategory(category);
-        setPlaceToLocalionTest(expectedPlace);
-        setPlaceToOpeningHoursTest(expectedPlace);
-    }
-
-    private Category setCategoryToPlaceTest() {
-        Category category = new Category();
-        category.setName("cafe");
-        category.setId(1L);
-        CategoryDto categoryDto = new CategoryDto();
-        categoryDto.setName("cafe");
-        categoryService.save(categoryDto);
-        return category;
-    }
-
-    private void setPlaceToOpeningHoursTest(Place expectedPlace) {
-        OpeningHours openingHours = new OpeningHours();
-        openingHours.setOpenTime(LocalTime.parse("10:30"));
-        openingHours.setOpenTime(LocalTime.parse("20:30"));
-        openingHours.setPlace(expectedPlace);
-        OpeningHours expectedOpeningHours = new OpeningHours();
-        openingHours.setId(1L);
-        openingHours.setOpenTime(LocalTime.parse("10:30"));
-        openingHours.setOpenTime(LocalTime.parse("20:30"));
-        openingHours.setPlace(expectedPlace);
-    }
-
-    private void setPlaceToLocalionTest(Place expectedPlace) {
-        Location location = new Location();
-        location.setAddress("test address");
-        location.setLat(45.456);
-        location.setLng(46.456);
-        location.setPlace(expectedPlace);
-        locationService.save(location);
-        Location expectedLocation = new Location();
-        expectedLocation.setId(1L);
-        location.setAddress("test address");
-        location.setLat(45.456);
-        location.setLng(46.456);
-        location.setPlace(expectedPlace);
+        when(categoryService.findByName(anyString())).thenReturn(any());
+        when(categoryService.save(categoryDto)).thenReturn(category);
+        when(placeAddDtoMapper.convertToEntity(placeAddDto)).thenReturn(placeEntity);
+        when(placeRepo.save(any())).thenReturn(new Place());
+        when(userService.findByEmail(anyString())).thenReturn(null);
+        when(service.findByLatAndLng(anyDouble(), anyDouble())).thenReturn(location);
+        when(service.findByLatAndLng(45.45, 45.456)).thenReturn(null);
+        when(service.save(location)).thenThrow(BadLocationRequestException.class);
+        when(openingHoursService.save(openingHours)).thenReturn(openingHours);
```

Comment:
strange test without "run" section?

seems like just mocked methods declaration and nothing more?